composer/loggers/mlflow_logger.py

```diff
@@ -162,7 +162,7 @@ def init(self, state: State, logger: Logger) -> None:
                 # Search for an existing run tagged with this Composer run.
                 assert self._experiment_id is not None
                 existing_runs = mlflow.search_runs(experiment_ids=[self._experiment_id],
-                                                   filter_string=f'tags.composer_run_name = "{state.run_name}"',
```

Comment:
do we need to search for the old key too? probably just to make no workloads get randomly broken?

Reply:
probably...backwards compatibility is such a concept